x-pack/test/security_solution_cypress/cypress/tasks/asset_criticality/common.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  OPEN_HOST_FLYOUT_BUTTON,
+  OPEN_USER_FLYOUT_BUTTON,
+  HOST_DETAILS_FLYOUT_ASSET_CRITICALITY_SELECTOR,
+  HOST_DETAILS_FLYOUT_ASSET_CRITICALITY_BUTTON,
+} from '../../screens/asset_criticality/flyouts';
+
+/**
+ * Find the first alert row in the alerts table then click on the host name to open the flyout
+ */
+export const expandFirstAlertHostFlyout = () => {
+  cy.get(OPEN_HOST_FLYOUT_BUTTON).first().click();
+};
+
+/**
+ * Find the first alert row in the alerts table then click on the host name to open the flyout
+ */
+export const expandFirstAlertUserFlyout = () => {
+  cy.get(OPEN_USER_FLYOUT_BUTTON).first().click();
+};
+
+/**
+ * Open the asset criticality accordion
+ */
+export const toggleAssetCriticalityAccordion = () => {
+  cy.get(HOST_DETAILS_FLYOUT_ASSET_CRITICALITY_SELECTOR).scrollIntoView();
+  cy.get(HOST_DETAILS_FLYOUT_ASSET_CRITICALITY_SELECTOR).should('be.visible').click();
```

Comment:
NIT: Cypress automatically performs different checks and actions before doing a click into an element when we are not forcing it as it is the case:

* Scroll the element into view.
* Ensure the element is not hidden.
* Ensure the element is not disabled.
* Ensure the element is not detached
* Ensure the element is not readonly.
* Ensure the element is not animating.
* Ensure the element is not covered
* Scroll the page if still covered by an element with fixed position.

So it should not be needed to do the the `scrollIntoView` or the implicit check on visibility.